### DISCUSSION TITLE
changing font and default size on Sleek 2.X

### DISCUSSION THREAD

**Discussion by Pelado-Mat** [ransome1/sleek#686] [Q&A] [answered]
Is there a way to change the font and size?  The zoom function kind of works, but I really want to change the font.

**Comment by ransome1** [comment, marked as answer]
> Is there a way to change the font and size? The zoom function kind of works, but I really want to change the font.

Hi @Pelado-Mat you can adjust the zoom level, which literally changes the font size. Plus of course all other elements, which align with the font size.

There is no way of changing the font itself and this is also not planned. 

However there is a way of injecting CSS into sleek. You might be able to play around with it and be able to change the font size and type. It is described here: https://github.com/ransome1/sleek/wiki/Custom-CSS

**Reply by Pelado-Mat** [reply]
Thanks! will try that!
